fixtures/user/user_fixture.py

```diff
@@ -515,7 +526,7 @@
 set_user_location_exists_invalid_location_response = {
     "errors": [
         {
-            "message": "The location supplied does not exist",
```

Comment:
maybe this test should be removed since now this error `The location supplied exists ` can no longer be reached with your changes, changing only the message error gives the impression that it is doing the same thing as the test giving the error message on line 489.

Reply:
I have removed the tests and fixtures that were no longer in use.